imqsauth/http.go

```diff
@@ -668,6 +734,7 @@ func httpHandlerSetUserGroups(central *ImqsCentral, w http.ResponseWriter, r *ht
 		}
 	}
 	*/
+	central.sendAll("auth:logout:" + identity)
```

Comment:
I don't think this message should be called "logout", because it's not true. Rather make the message convey exactly what is actually happening. In this case, it is "permissions_changed". Leave it up to the listener to decide what he wants to do with that information.